doc/api/fs.md

```diff
@@ -2495,6 +2687,7 @@ The following constants are meant for use with the [`fs.Stats`][] object's
 [`stat()`]: fs.html#fs_fs_stat_path_callback
 [`util.inspect(stats)`]: util.html#util_util_inspect_object_options
 [`WriteStream`]: #fs_class_fs_writestream
+[URL]: url.html#url_the_whatwg_url_api
```

Comment:
Hi @TimothyGu what alphabetical order ? I was trying to figure it out if there was something like that when I included that change. However i couldn't guess what the logic was as list items seems a bit melted. 

Does internal vs external links matters in the order ?
Does functions / classes matters ?

If you can describe me the logical structure beneath the list I will be happy to sort that :)